README.md

```diff
@@ -554,46 +590,55 @@ openebs-jiva-operator-55cb6679b6-gjjlj              1/1     Running   0
+
 ```shell
 kubectl taint node rollnet-tpi-1 node-role.kubernetes.io/control-plane:NoSchedule-
 kubectl taint node rollnet-tpi-2 node-role.kubernetes.io/control-plane:NoSchedule-
 kubectl taint node rollnet-tpi-3 node-role.kubernetes.io/control-plane:NoSchedule-
 ```
-#
-Since Jiva assumes iscisd to be running natively on the host and not as a Talos extension service, we need to modify the CSI node daemonset to enable it to find the PID of the iscsid service. The default config map used by Jiva also needs to be modified so that it can execute iscsiadm commands inside the PID namespace of the iscsid service.
-#
+
+Since Jiva assumes iscsid to be running natively on the host and not as a Talos extension service, we need to modify the CSI node daemon set to enable it to find the PID of the iscsid service. In addition, Jiva also needs to modify the default config map to execute iscsiadm commands inside the PID namespace of the iscsid service.
+
 Create a file named `configmap.yaml` with the following contents (an example is also stored in the manifest directory of this repo):
 
 ```yaml
 apiVersion: v1
 kind: ConfigMap
 metadata:
-  labels:
-    app.kubernetes.io/managed-by: pulumi
   name: openebs-jiva-csi-iscsiadm
   namespace: openebs
+  labels:
+    app.kubernetes.io/managed-by: pulumi
 data:
   iscsiadm: |
     #!/bin/sh
+    set -euo pipefail
+
+    # Find the process ID of the iscsid daemon
     iscsid_pid=$(pgrep iscsid)
 
-    nsenter --mount="/proc/${iscsid_pid}/ns/mnt" --net="/proc/${iscsid_pid}/ns/net" -- /usr/local/sbin/iscsiadm "$@"
+    # Enter the namespaces of the iscsid process and execute iscsiadm with given arguments
+    nsenter --mount=/proc/${iscsid_pid}/ns/mnt --net=/proc/${iscsid_pid}/ns/net -- /usr/local/sbin/iscsiadm "$@"
 ```
 
 Apply this file to replace the existing configmap:
+
 ```shell
 kubectl --namespace openebs apply --filename configmap.yaml
 ```
 
 Next, the Jiva CSI daemonset needs to be run with `hostPID: true` so it can find the PID if the iscsid service we installed with the extension:
+
 ```shell
 kubectl --namespace openebs patch daemonset openebs-jiva-csi-node --type=json --patch '[{"op": "add", "path": "/spec/template/spec/hostPID", "value": true}]'
 ```
+
 ## Test the storage
-If everything installed correctly, we should now be able to use our distributed storage with workloads.  To test this you can use the following PVC and Deployment manifests (examples are also stored in the manifest directory of this repo):
 
+If everything installed correctly, we should now be able to use our distributed storage with workloads. To test this you can use the following PVC and Deployment manifests (examples are also stored in the manifest directory of this repo):
 
 Create a file named `deployment-test.yaml` with the following contents:
+
```

Comment:
I like the changes you made to the pvc and configmap, but I would rather leave the deployment more basic. Also, since we're not setting anything up to serve healthz in the busybox container.  The healthz directory is built into the k8s.gcr.io/liveness image and used in many examples for readiness and liveness probes so I'm assuming that's where you pulled it from. Same goes for the manifest file.